myscript.js

```diff
@@ -0,0 +1,35 @@
+$(document).ready(function(){
+    var femaleName1 = prompt("Enter a female name (1 of 2)");
+        $('#female1_01').text(femaleName1);
+        $('#female1_02').text(femaleName1);
+        $('#female1_03').text(femaleName1);
```

Comment:
Very nice! I like that you've used the anonymous function we learned to contain this code that gathers the values you need and then you assign them straight away with jQuery. Again, excellent formatting. Very clean and easy to read.